java/test/jmri/ProgrammerScaffold.java

```diff
@@ -22,6 +22,7 @@
  *
  * @see jmri.Programmer
  * @author Bob Jacobsen Copyright (C) 2008, 2014
+ * @author Andrew Crosland Copyright (C) 2008, 2021
```

Comment:
It would be good to remove this comment now when the new `readCV` method is removed from this class.